src/scripts/main.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
 function waitFor(element, eventName) {
-  // write your code here
+  return new Promise((resolve, reject) => {
+    element.addEventListener(eventName, () => {
+      const message = `It was ${eventName} on the element: ${element.nodeName}, id: ${element.id}`;
+
+      resolve(message);
+    });
+  });
```

Comment:
The `waitFor` function adds an event listener for the specified event on the element. However, once the event is triggered and the promise is resolved, the event listener remains active. This could lead to performance issues if the event is triggered multiple times. Consider removing the event listener after it has been triggered once, using `element.removeEventListener()`.